aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -104,6 +104,20 @@ def validate_parser(parser_name, _):
             return f'invalid parser specified: {exception}'
 
 
+def validate_additional_retrieve_list(additional_retrieve_list, _):
+    """Validate the additional retrieve list.
+
+    :return: string with error message in case the input is invalid.
+    """
+    import os
+
+    if additional_retrieve_list is plumpy.UNSPECIFIED:
+        return
```

Comment:
`_` is just a placeholder, the signature requires this argument, but since we are not using it, I just use an underscore.

Reply:
`_` is just a placeholder, the signature requires this argument, but since we are not using it, I just use an underscore. The `plumpy.UNSPECIFIED` is there because of a bug in `plumpy`. I have fixed this in `0.17.0` but we don't depend on this here yet, so I cannot remove it yet. Basically, `plumpy` calls a port validator even if no value for it is specified and so it will pass the default "none" value, which is the `plumpy.UNSPECIFIED` constant.